supervisor/options.py

```diff
@@ -520,6 +519,7 @@ def realize(self, *arg, **kw):
         self.serverurl = None
 
         self.server_configs = sconfigs = section.server_configs
+        self.directories = section.directories
```

Comment:
so section would be configparser section which defaults to None if not present ?